doc/adr/0009-make-offline-mode-file.md

```diff
@@ -19,26 +19,67 @@ SQLite plugin for Flutter. Supports iOS, Android and MacOS.
 
 Usage 
 
-- Opening a database
-```
-Future<Database> openDatabase(String path, {OpenDatabaseOptions options});
-```
-- Insert record
+- Table Document: Help save documents when it make available offline in MySpace
+
 ````
-Future<int> insert(String table, Map<String, dynamic> values)
+Table Document {
+  documentId TEXT PRIMARY KEY
+  description TEXT
+  creationDate Integer
+  modificationDate Integer
+  expirationDate Integer
+  ciphered Integer
+  name TEXT
+  size Integer
+  sha256sum TEXT
+  hasThumbnail Integer
```

Comment:
are you sure it is an `Integer`?

Reply:
Yes, sqlite not support boolean type so we can use integer with value `0` & `1` for it